lib/client-assets.php

```diff
@@ -872,34 +872,83 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		);
 	}
 
+	/**
```

Comment:
As by the previous note, this function doesn't exist as of #13569.

All new functionality should extend the block editor as if it were any other plugin extending the core block editor.